packages/protocol/package.json

```diff
@@ -17,16 +22,17 @@
     "flatten:airnode": "hardhat flatten contracts/AirnodeRrp.sol > AirnodeRrp.flat.sol && yarn run flatten:delete-license-lines",
     "flatten:delete-license-lines": "sed --in-place '/SPDX-License-Identifier/d' *.flat.sol"
   },
-  "dependencies": {
```

Comment:
A lot of the dependencies are under `dependencies` and not `devDependencies` because of how the deployer Docker container was being built in pre-alpha. We will move away from that so this should be done in a few other `package.json`s. Just giving some context.